xcube_clms/store.py

```diff
@@ -98,3 +146,16 @@ def get_search_params_schema(
         cls, data_type: DataTypeLike = None
     ) -> JsonObjectSchema:
         pass
+
+    def preload_data(self, *data_ids: str, **preload_params):
+        return self._clms.preload_data(*data_ids, **preload_params)
+
+    @classmethod
+    def get_preload_params_schema(cls):
+        params = dict(
```

Comment:
`data_ids` is not in `preload_parms`. But you could add a flag if visualization is shown or not. I'd say default to True. In notebook, visualization is useful, but in a script it can be annoying. 
Then you can already validate to the preload-params schema when calling `preload_data`.